app/test/integration-test/pom.xml

```diff
@@ -295,17 +295,6 @@
     <testResources>
       <testResource>
         <directory>src/test/resources</directory>
-        <filtering>true</filtering>
```

Comment:
@christophd Are these filters needed? for some reason, with these filters, any jar I am using to test is converted by maven in a zip and the integration is failing.

Reply:
I think we need to add an exclude for `*.jar` files when those live in the `src/test/resources` folder. Initially we have used integration exports as `.zip` but now we are using the pure `model.json`. So I think we could also remove that part from the `pom.xml`